frontend/src/nc/pages/Dashboard.jsx

```diff
@@ -80,7 +80,11 @@ const Dashboard = () => {
           setViewHelp={setViewHelp}
         />
       </div>
-      <div className="flex flex-1 flex-col overflow-hidden">
+      <div className="flex flex-1 flex-col overflow-hidden" style={
+        {
+          overflow: "auto",
```

Comment:
isn't there a built-in construct that handles this case? Do we need to explicitly add an inline style attribute?

Reply:
Done